source/macros.tex

```diff
@@ -337,8 +337,24 @@
 
 %% Change descriptions
 \newcommand{\diffdef}[1]{\hfill\break\textbf{#1:}\space}
-\newcommand{\diffref}[1]{\pnum\textbf{Affected subclause:} \ref{#1}}
-\newcommand{\diffrefs}[2]{\pnum\textbf{Affected subclauses:} \ref{#1}, \ref{#2}}
+\RequirePackage{expl3,xparse}
```

Comment:
I'd prefer this to be \usepackage items in std.tex so that the list of required packages is clearly visible where a (LaTeX 2) person would expect it.

Reply:
@jensmaurer Done!